api-keys-rate-limits.md

```diff
@@ -6,7 +6,7 @@ To use the Mapzen Search service, you must first get a developer API key. Sign i
 
 1. Go to https://mapzen.com/developers.
 2. Sign in with your GitHub account. If you have not done this before, you need to agree to the terms first.
-3. Create a new key for Mapzen Search, and optionally, give it a name so you can remember the purpose of the project.
+3. Create a new key and optionally, give it a name so you can remember the purpose of the project.
```

Comment:
thanks. moved it around (which we now should do on all the instances...need smart content reuse variables!)